sdk/storage/Azure.Storage.Files.Shares/src/autorest.md

```diff
@@ -0,0 +1,10 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
```

Comment:
`azure-sdk-for-net/sdk/storage/Azure.Storage.Files.Shares/src/dotnet build /t:GenerateCode`